src/js/button.js

```diff
@@ -1,3 +1,8 @@
+import Component from './component';
+import * as Lib from './lib';
```

Comment:
Sounds like we'd have a utils directory for the broken out libs, which will be individual modules that you'll import individually. That sounds good to me. We still need to nail down exactly how we're exposing these functions to plugin authors, which may not affect this exactly but could. #1232